tests/safety/test_ford.py

```diff
@@ -96,13 +101,23 @@ def _acc_button_msg(self, button: int, bus: int):
     return self.packer.make_can_msg_panda("Steering_Data_FD1", bus, values)
 
   def test_steer_allowed(self):
-    self.safety.set_controls_allowed(1)
-    self.assertTrue(self._tx(self._tja_command_msg(1)))
-    self.assertTrue(self.safety.get_controls_allowed())
-
-    self.safety.set_controls_allowed(0)
-    self.assertFalse(self._tx(self._tja_command_msg(1)))
-    self.assertFalse(self.safety.get_controls_allowed())
+    path_offsets = np.arange(-5.12, 5.11, 1).round()
```

Comment:
how long does this test take?

Reply:
0.01 ms to 600 ms. Now 100 ms with the latest commit